README.md

```diff
@@ -13,3 +13,5 @@ The server expects the following environment variables to be set:
 * `FICAI_DB_USERNAME` is the user name for DB access
 * `FICAI_DB_PASSWORD` is the password for the abovementioned user
 * `FICAI_DB_DATABASE` is the name of the database in which the server's tables must be present
+* `FICAI_PWD_PEPPER` is the pepper value for password hashes. Read more at https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#peppering
```

Comment:
A comment somewhere with best practices on how long this should be, and that it's expecting an unpadded base64 value would be nice but not crucial.